recipes/bzip2/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "1.0.6" %}
+
+package:
+  name: bzip2
+  version: {{ version }}
+
+source:
+  fn: bzip2-{{ version }}.tar.gz
+  url: http://www.bzip.org/{{ version }}/bzip2-{{ version }}.tar.gz
+  md5: 00b516f4704d4a7cb50a1d97e6e8e15b
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+  skip: true  # [win and py36]
+
+requirements:
+  build:
+    - toolchain
+    - python {{ os.environ.get('PY_VER', '') + '*' }}  # [win]
```

Comment:
So we don't need this in `build` as this happens behind the scenes. It is only needed in `test`. For history, please see the linked issue.

xref: https://github.com/conda/conda-build/issues/835